### ISSUE TITLE
Bruker Parameter PVM_NRepetitions not accounted for in reconstruction.

### ISSUE THREAD

**Issue by benjhardy** [remmi-toolbox/remmi-matlab#1]
Seems to me that PVM_NRepetitions is not accounted for. It is confusing because PVM_NAverages also exists.
I did a simple FLASH imaging experiment and whenever Repetitions (PVM_NRepetitions) was set to more than 1, the reconstruction failed with this error:
```
Error using reshape
To RESHAPE the number of elements must not change.

Error in remmi.vendors.loadBruker (line 177)
data = reshape(data,encmatrix(1),encmatrix(2),encmatrix(3),length(echotimes),...

Error in remmi.vendors.BrukerPV/load (line 168)
            raw = remmi.vendors.loadBruker(expPath,pars.methpars);

Error in remmi.recon (line 68)
    [dset.img{n},dset.labels{n},dset.pars{n}] = study.load(num2str(exps{n}),options);
```
Whenever  PVM_NAverages is  1 or more the reconstruction is successful.

When PVM_NRepetitions is N, (where N>1) the data array is N times larger. and thus fails

**Comment by benjhardy** [comment]
I think this has to do with the phase encode extending beyond the returned image.
The image data collected maybe within the phase encoding dimensions.

**Comment by kevinharkins** [comment]
Fixed w/ commit #84ea9d8
